src/mergeProp.js

```diff
@@ -0,0 +1,28 @@
+import { curry } from 'ramda';
+import mergePath from './mergePath';
+
+/**
+ * Merges user provided data (subject) with some object (target)
+ * inside other object under corresponding property.
+ *
+ * @func mergeProp
+ * @memberOf RA
+ * @since {@link https://char0n.github.io/ramda-adjunct/1.18.0|v1.18.0}
+ * @category Object
+ * @sig [k] -> {a} -> {k: {a}} -> {k: {a}}
+ * @see {@link RA.mergeProp|mergeProp}
+ * @param {!Array} prop The prop to object to merge with
```

Comment:
pleare rename to `p` to be consistent with ramda, `mergeProps` and typescript type signatures.

Also the type is wrong, should be 
```javascript
@param {string} p The property of the destination object